src/components/PlanRoom/RoomTables/Item.tsx

```diff
@@ -0,0 +1,184 @@
+import { useCallback, useState, SetStateAction } from 'react';
+import { Box, Textarea } from '@chakra-ui/react';
+import update from 'immutability-helper';
+import { useDrop, XYCoord, useDrag } from 'react-dnd';
+import ItemTypes from './ItemTypes';
+import { TypeItem, TableType } from './Types';
+import SaveBoard from '../SaveRoomTables/SaveBoard';
+
+let changeValueTable: Function;
+let changeValueTableBorderRadius: Function;
+let newTable: Function;
+let save: Function;
```

Comment:
You should not declare variables in "global scope" of this file.
Use `state` or another (proper) hooks to do that.

Reply:
I will leave it at that for now